orderbook/usecase/orderbook_usecase.go

```diff
@@ -137,6 +137,63 @@ func (o *OrderbookUseCaseImpl) ProcessPool(ctx context.Context, pool sqsdomain.P
 	return nil
 }
 
+// GetActiveOrdersStream implements mvc.OrderBookUsecase.
+func (o *OrderbookUseCaseImpl) GetActiveOrdersStream(ctx context.Context, address string) <-chan orderbookdomain.ActiveOrder {
```

Comment:
Let's add some tests for this.

Reply:
I have covered this method with tests [here](https://github.com/osmosis-labs/sqs/pull/518/files#diff-5fa55bd6410efa835405589edaffbb41ba95f810344564647d2d711e8bc15b67R242)